core/Application.php

```diff
@@ -6,12 +6,32 @@
  * @author Katsuhiro Ogawa <fivestar@nequal.jp>
  */
 abstract class Application
+abstract class Application
```

Comment:
ERROR: Each class must be in a namespace of at least one level (a top-level vendor name)